### DISCUSSION TITLE
mic icon

### DISCUSSION THREAD

**Discussion by noocyn** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#283] [Q&A]
I noticed I had to press the mic icon every time before I make a command. Is there a way to wake it up by saying 'ok google' without pressing the mic icon?

**Comment by esau-morais** [comment]
This feature is already in the roadmap.

**Comment by sumeshir26** [comment]
@noocyn This has been implemented in the build version and will be released soon...
